chapters/VK_KHR_surface/wsi.adoc

```diff
@@ -1020,6 +1020,41 @@ present barrier feature is not obtainable for this surface.
 include::{generated}/validity/structs/VkSurfaceCapabilitiesPresentBarrierNV.adoc[]
 --
 endif::VK_NV_present_barrier[]
+
+ifdef::VK_EXT_present_timing[]
+[open,refpage='VkPresentTimingSurfaceCapabilitiesEXT',desc='Structure describing present timing capabilities of a surface',type='structs']
+--
+The sname:VkPresentTimingSurfaceCapabilitiesEXT structure is defined as:
+
+include::{generated}/api/structs/VkPresentTimingSurfaceCapabilitiesEXT.adoc[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:presentTimingSupported indicates whether querying presentation
+    timestamps is supported for a swapchain created from
+    slink:VkPhysicalDeviceSurfaceInfo2KHR::pname:surface.
+  * pname:presentAtAbsoluteTimeSupported indicates whether a swapchain
+    created from slink:VkPhysicalDeviceSurfaceInfo2KHR::pname:surface
+    supports presenting images with absolute times using
+    slink:VkPresentTimeEXT::pname:targetPresentTime.
+  * pname:presentAtRelativeTimeSupported indicates whether a swapchain
+    created from slink:VkPhysicalDeviceSurfaceInfo2KHR::pname:surface
+    supports presenting images with relative times using
+    slink:VkPresentTimeEXT::pname:minPresentDuration.
```

Comment:
Should this be "presentDuration" or targetPresentTime? I can't find any other instances of "minPresentDuration"